fileio/private/ft_hastoolbox.m

```diff
@@ -219,6 +220,8 @@
     dependency = {'spm', get_spm_version()>=12, get_spm_version()<95};
     %This is to avoid crashes when trying to add SPM to the path
     fallback_toolbox = 'SPM12';
+    case 'MAYO_MEF' % MEF 2.1 and MEF 3.0
+        dependency = {};
```

Comment:
please add some functions to this list, such that the ft_hastoolbox can determine whether it is on the path or not. Please search for examples of this elsewhere in the FT code.